include/seqan3/range/decorator/gap_decorator.hpp

```diff
@@ -356,6 +502,10 @@ class gap_decorator
     //!\brief The const_iterator equals the iterator type. Since no references are ever returned and thus the underlying
     //!        sequence cannot be modified through the iterator there is no need for const.
     using const_iterator = iterator;
+    //!\brief The pseudo random access iterator type (provides functionality but **not in constant time**).
+    using iterator_ra = gap_decorator_iterator_ra;
+    //!\brief The const_iterator_ra equals the iterator_ra type.
+    using const_iterator_ra = iterator_ra;
```

Comment:
Yes, because the decorator can never be modified over the iterators, so every iterator is a const iterator anyway. The reason: First a decorator only "decorates another sequence with gaps", so it should not be able to alter the underlying sequence. Thus `*it = 'A'_dna4` would never be allowed and for consistency `*it = gap{}` is also not allowed but the decorator mus be modified through "insert_gap" functions.